src/app/integrations/edit-page/flow-view/flow-view.component.scss

```diff
@@ -135,10 +135,8 @@ $flow-view-container-min-height: 0;
 
   .steps {
     position: relative;
-    //height: 150px;
-    height: 100%;
     flex-wrap: nowrap;
-    min-height: 250px;
+    min-height: 150px;
```

Comment:
Sorry about that, just wanted it to be spaced out more on a larger screen.. maybe we can address that later on.